test/saml2.coffee

```diff
@@ -459,6 +463,46 @@ describe 'saml2', ->
         assert.deepEqual response, expected_response
         done()
 
+    it 'correctly parses an AuthnStatement with no session_index', (done) ->
```

Comment:
Awesome to have a test 👌 . If you wanted to do the extra credit it would be great to have a test to ensure that it continues to _fail_ to parse if you don't set the `require_session_index` flag.